src/components/organisms/AdminVenueView/components/RoomCard/RoomCard.scss

```diff
@@ -0,0 +1,65 @@
+@import "scss/constants.scss";
+
+.RoomCard {
+  &__card {
+    border-radius: $border-radius--lg;
+    padding: $spacing--md;
+    width: min(calc(100% - 2 * #{$spacing--xs}), 30em);
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    min-width: 0;
+    word-wrap: break-word;
+    background-color: $room-card-background;
+    background-clip: border-box;
+  }
+
+  &__header {
+    display: grid;
+    grid-template: 32px 32px 2em / 64px 1fr 6em;
+    place-items: center center;
+    column-gap: 1em;
```

Comment:
again `rem` vs `em`? I'd vote for rem, but not super against of using `em`.

Reply:
I'd vote for a third option, which I think already has some in a different PR https://github.com/sparkletown/sparkle/pull/1950#discussion_r685163294

Using these ```scss
// Relative font sizes
$font-size--document: 1rem;
$font-size--parent: 1em;
```
makes the intent clearer and also opens the door for creating scaling functions like
```scss
@function times-of-parent($scale){
  @return $scale * $font-size--parent;
}
@function times-of-document($scale){
  @return $scale * $font-size--document;
}
```
that can be used for margins and paddings relative to font spacing